draft-ietf-rats-architecture.md

```diff
@@ -1317,3 +1317,160 @@ use beyond the period for which it deems the Attestation Result to remain
 valid.  Thus, if the Attestation Result sends a validity lifetime
 in terms of time(RX)-time(RG), then the Relying Party can check
 `time(OP) - time(ER) < time(RX)-time(RG)`.
+
+# Appendix B: System Examples
+
+(Rough first draft to see if there is interest)
+
+## System Example with OS Vendor as Main Driver
+
+Three entities (companies / organizations) participate in this examples.
+
+The operating system vendor is at the center. It sets the architecture and the level of security achieved. It performs the following:
+* Generates large quantities of ECDSA key pairs to be used as per-device attestation keys
+* Signs the public keys using an attestation PKI and root that they control creating per-device certificates.
+* Securely sends the private keys and certificates to the device OEM in batches to be used in manufacturing
+* Authors the device SW that is the Attester and ships it to the OEMs. This is a part of the OS.
```

Comment:
"This is part of the OS" is probably incorrect for many situations.

Reply:
I think it is reasonable in this example.  Think of an OS vendor like Google supplying both the TEE which has the attester and Android.  

In some cases the TEE is made by another, but adding that would complicate the example a lot for little gain.